from exc.MyException import CustomException


class DomainException(CustomException):
    pass
